lte/gateway/c/core/oai/tasks/nas5g/include/ies/M5GQOSRules.hpp

```diff
@@ -22,20 +22,28 @@ class NewQOSRulePktFilter {
   uint8_t pkt_filter_dir : 2;
   uint8_t pkt_filter_id : 4;
   uint8_t len;
-  uint8_t contents;
+
+  // Size of the content can be max of packet_filter_contents_t
+  uint8_t contents[2 * TRAFFIC_FLOW_TEMPLATE_IPV6_ADDR_SIZE + sizeof(uint8_t)];
+
   NewQOSRulePktFilter();
   ~NewQOSRulePktFilter();
 };
 
 // QOSRule Class
 class QOSRule {
  public:
+#define QOS_ADD_RULE_MIN_LEN 3
+#define QOS_DEL_RULE_MIN_LEN 1
   uint8_t qos_rule_id;
   uint16_t len;
   uint8_t rule_oper_code : 3;
+#define QOS_RULE_DQR_BIT_SET 0x1
```

Comment:
Can you move it up to the other 2 #defines?

Reply:
its been moved..